src/Components/Publishing/Icon/IconVideoMute.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react"
+
+export class IconVideoMute extends Component<any, null> {
```

Comment:
Might make sense to include color prop on all icons, especially if you're thinking to package up.